app/src/main/java/com/riyas/myapplication/LoginController.kt

```diff
@@ -0,0 +1,8 @@
+package com.riyas.myapplication
+
+class LoginController {
+
+    fun doLogin(username: String, password: String) {
```

Comment:
Is this name okay for you guys? @akhilraj-c

Reply:
Yes